Added 12 new Q&A pairs to the Support knowledgebase section (#1081)
wandb/docs: content/support/how_can_i_delete_multiple_runs_efficiently_instead_of_choosing_them_one_by_one.md

---

> title: How can I delete multiple runs efficiently instead of choosing them one by one?

Reviewer: ```suggestion
title: How can I delete multiple runs in bulk instead of one at a time?
```

toc_hide: true
type: docs
tags:
  - projects
  - runs
---

> Use the Public API to delete multiple runs efficiently:

Reviewer: ```suggestion
Use the [public API]({{< relref "/ref/python/public-api/api.md" >}}) to delete multiple runs in a single operation:
```
